api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -303,8 +339,12 @@ type FelixConfigurationSpec struct {
 	// server. For example, if a significant security vulnerability has been discovered in the version of Calico being used. [Default: true]
 	UsageReportingEnabled *bool `json:"usageReportingEnabled,omitempty"`
```

Comment:
bool does not need the kubebuilder annotation?

Reply:
I remember testing this and it only accepted "true" and "false" for bool. I think kubebuilder already handles that based on the type.